Main.elm

```diff
@@ -1,7 +1,288 @@
-module Main exposing (..)
+port module Main exposing (..)
 
 import Html
+import Html.Attributes
+import Html.Events
+import Json.Decode as Decode
+import Json.Encode as Encode
+import Kinto
+
+
+-- Settings
+-- If you change the kintoServer there you want to change it in ports.js too.
+
+
+kintoServer =
+    "https://kinto.dev.mozaws.net/v1/"
+
+
+bucket =
+    "stepfunction"
+
+
+collection =
+    "manual_steps"
+
+
+
+-- Model
+
+
+type alias Flags =
+    { email : Maybe String
+    , bearer : Bearer
+    }
+
+
+type alias Model =
+    { email : String
+    , bearer : Bearer
+    , records : Maybe (List Record)
+    , error : Maybe String
+    }
+
+
+type alias Bearer =
+    Maybe String
+
+
+type alias RecordId =
+    String
+
+
+type alias Record =
+    { id : RecordId
+    , last_modified : Int
+    , subject : Maybe String
+    , stateMachineArn : String
+    , activityArn : String
+    , status : Maybe String
+    }
+
+
+
+-- Msg
+
+
+type Msg
+    = NewEmail String
+    | Authenticate
+    | LoadRecords
+    | FetchRecordsResponse (Result Kinto.Error (List Record))
+    | Logout
+
+
+
+-- Init
+
+
+init : Flags -> ( Model, Cmd Msg )
+init flags =
+    let
+        model =
+            { email = Maybe.withDefault "" flags.email
```

Comment:
Sorry my answer was silly... I can keep the email here a a Maybe and handle the with default directly in the input value.